validator/imported_test.go

```diff
@@ -2,6 +2,7 @@ package validator_test
 
 import (
 	"fmt"
+	"github.com/vektah/gqlparser/v2/validator"
```

Comment:
Would you mind putting this import down with the non-standard library imports?

Reply:
@StevenACoffman 
Thanks! I fixed it like
```go
import (
	"testing"

	"github.com/stretchr/testify/require"

	"github.com/vektah/gqlparser/v2"
	"github.com/vektah/gqlparser/v2/ast"
	"github.com/vektah/gqlparser/v2/parser"
	"github.com/vektah/gqlparser/v2/validator"
	"github.com/vektah/gqlparser/v2/validator/rules"
)
```